coralproject/talk
# remove express dep from mailer service

## What does this PR do?

Initially I set up the mailer service with a dependency injection using the express app to render the email templates. This meant that you needed a request object to send an email. No more!

I updated the mailer to just load the templates as strings, and use `_.template` which works the same way.

## How do I test this PR?

- sign up for a new user
- you should get an email confirmation that works.
- maybe try to reset your password and make sure that still works too
